src/deepsparse/pipeline.py

```diff
@@ -176,29 +177,34 @@ def __init__(
 
         self._batch_size = self._batch_size or 1
 
-    def __call__(self, *args, **kwargs) -> BaseModel:
+    def __call__(self, monitoring: bool = False, *args, **kwargs) -> BaseModel:
```

Comment:
a kwargs should go after `*args` in this case
```suggestion
    def __call__(self, *args, monitoring: bool = False, **kwargs) -> BaseModel:
```